nemo_aligner/models/nlp/gpt/megatron_gpt_critic.py

```diff
@@ -15,7 +15,7 @@
 from enum import Enum
 
 import torch
-from apex.transformer.pipeline_parallel.utils import _reconfigure_microbatch_calculator, get_num_microbatches
+from megatron.core.num_microbatches_calculator import _reconfigure_microbatch_calculator, get_num_microbatches
```

Comment:
mostly looks good to me. is there a way for them to not have a private function? can we make it an API they support?

because aligner needs to run inference at potentially a different gbs/mbs

Reply:
I've updated the mcore MR to use not use the protected syntax and updated this PR. Will see if they accept that revision